### PR TITLE
#1-4 Task. Sergei.Dem

### PR DESCRIPTION
**Pull request by Trickster69**
https://travis-ci.com/github/Trickster69/js-assignments/builds/175551494

### CODE REVIEW

File: task/01-strings-tasks.js [hunk 1]
```diff
@@ -221,8 +223,15 @@ function getRectangleString(width, height) {
...
-    throw new Error('Not implemented');
+  return str.replace(
+    /[A-Z]/gi,
+    (i) =>
+      "NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm"[
+        "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz".indexOf(i)
+      ]
+  );
 }
+console.log(encodeToRot13("hello"));
```
Comment (zhvirblis) [inline task/01-strings-tasks.js]: Please remove all logs

**Review by zhvirblis — COMMENTED** [review]
the PR has redundant changes in styling, spaces, comments....
Please follow the simple rules in the future:
 - do not add any non logic changes to code if review is required
 - if the changes in styling are important - you can do it as separate PR with styling only changes to avoid noise in the main PR

I'll mark the PR as draft. Please click "ready for review" when it will be fixed.

**Review by zhvirblis — COMMENTED** [review]
Please remove redundant changes(like spaces and quotes)

**Review by KostyaKov — COMMENTED** [review]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**